mlflow/models/docker_utils.py

```diff
@@ -107,7 +107,7 @@ def _build_image(image_name, entrypoint, mlflow_home=None, custom_setup_steps_ho
         _logger.info("Building docker image with name %s", image_name)
         os.system("find {cwd}/".format(cwd=cwd))
         proc = Popen(
-            ["docker", "build", "-t", image_name, "-f", "Dockerfile", "."],
+            ["docker", "build", "-t", image_name, "-f", "Dockerfile", ".", "--platform=linux/amd64"],
```

Comment:
```suggestion
[
                "docker",
                "build",
                "-t",
                image_name,
                "-f",
                "Dockerfile",
                ".",
                # <Insert comment on why we need this>
                "--platform=linux/amd64",
            ],
```

@murilommen Thanks for the contribution! Could you insert a comment on why we need `--platform=linux/amd64`?

Reply:
Done!

Reply:
Thanks!